modules/openapi-generator/src/main/resources/dart-dio/analysis_options.mustache

```diff
@@ -1 +1,6 @@
-analyzer:
\ No newline at end of file
+analyzer:
+  language:
+    strict-inference: true
+    strict-raw-types: true
+  strong-mode:
+    implicit-dynamic: false
```

Comment:
i see that there's no `implicit-casts:false` in this one, maybe thats the difference between mine and your usage?

Reply:
You are right, I just commented on your PR.